main.go

```diff
@@ -78,6 +81,13 @@ func main() {
 			"the manager will watch and manage resources in all namespaces")
 	}
 
+	// setting privileged pod security labels to operator ns
+	err = addPodSecurityPriviledgedLabels(watchNamespace)
```

Comment:
Handle case where watchNamespace is `""` as returned from
https://github.com/openshift/oadp-operator/blob/85a0a7f04652ad79668ff70772a39d2a2db91a7d/main.go#L180

Reply:
Maybe exit the function early with an error instead of trying to process `""`

Reply:
@kaovilai PTAL, update the PR https://github.com/openshift/oadp-operator/pull/763/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R189